src/modules/gateway/gateway.controller.ts

```diff
@@ -0,0 +1,25 @@
+import { Body, Controller, Post, UseGuards } from '@nestjs/common';
+import { ApiOkResponse, ApiOperation, ApiTags } from '@nestjs/swagger';
+import { Public } from '../auth/decorators/public.decorator';
+import { GatewayUseCases } from './gateway.usecase';
+import { InitializeWorkspaceDto } from './dto/initialize-workspace.dto';
+import { GatewayGuard } from '../auth/gateway.guard';
+
+@ApiTags('Gateway')
+@Controller('gateway')
+export class GatewayController {
+  constructor(private gatewayUseCases: GatewayUseCases) {}
+
+  @Post('/workspaces')
+  @ApiOperation({
+    summary: 'Initiates a workspace',
+  })
+  @ApiOkResponse({ description: 'Returns the workspace created' })
+  @Public()
```

Comment:
This is just to skip the globalGuard(AuthGuard) that tries to verify an user based token.